### DISCUSSION TITLE
Make Resources based on Relation

### DISCUSSION THREAD

**Discussion by OmarASharaf** [filamentphp/filament#2254] [Help]
I have a model called 'project' that has many relations and I don't want to put all the relations on the view or edit page because I have a lot of relations,
what I want to do is to make a Resources page that takes the id of the project from the URL route and shows the data based on that id

is that passable to do?

**Comment by danharrin** [comment]
You can conditionally display managers using the `canViewForRecord()` method on the manager - which returns true or false based on if the manager is visible for that record.
